gulpfile.js

```diff
@@ -200,8 +203,8 @@ gulp.task('postcss', 'build postcss files', function() {
     require('postcss-import')(),
     require('autoprefixer')(),
     require('postcss-calc')(),
-    require('postcss-color-function')(),
     require('postcss-custom-properties')(),
+    require('postcss-color-function')(),
```

Comment:
We aren't adding this, only changing its order in the list. It must be processed after `custom-properties` to enable statements like `color(var(--blue) a(0.5))`.